packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -20,22 +21,23 @@ def helm_handler(event, context):
 
     # resource properties
     cluster_name = props['ClusterName']
-    role_arn     = props['RoleArn']
-    release      = props['Release']
-    chart        = props['Chart']
-    version      = props.get('Version', None)
-    wait         = props.get('Wait', False)
-    timeout      = props.get('Timeout', None)
-    namespace    = props.get('Namespace', None)
-    repository   = props.get('Repository', None)
-    values_text  = props.get('Values', None)
+    role_arn = props['RoleArn']
```

Comment:
Please avoid aesthetic changes, those make the code review harder.

Any reason spacing was changed?